packages/conf-llvm/conf-llvm.11.1.0/files/configure.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+version=${1/%.?.?/}
+
+if hash brew 2>/dev/null; then
+    brew_llvm_config="$(brew --cellar)"/llvm*/${version}*/bin/llvm-config
+fi
+
+shopt -s nullglob
+for llvm_config in llvm-config-$version llvm-config-${version}.0 llvm-config${version}0 llvm-config-mp-$version llvm-config-mp-${version}.0 llvm${version}-config llvm-config-${version}-32 llvm-config-${version}-64 llvm-config $brew_llvm_config; do
```

Comment:
Note that I moved the brew_llvm_config entry to the end so that if someone has one version of llvm installed via brew and has another earlier in PATH, the PATH one wins.